### PR TITLE
chore(docs): update unpublish docs with both commands, removing policy info from cli docs, and added reference to unpublish policy docs

### PR DESCRIPTION
**Pull request by nomadtechie**
- Updating docs unpublish docs to with examples of both unpublish commands
- Removing policy info from cli docs to centralize policy in policy docs
- Added reference to unpublish policy docs

### CODE REVIEW

File: docs/content/cli-commands/npm-unpublish.md [hunk 1]
```diff
@@ -10,19 +10,21 @@ description: Remove a package from the registry
 
 ### Synopsis
 
+#### Unpublishing a single version of a package
+
 ```bash
-npm unpublish [<@scope>/]<pkg>[@<version>]
+npm unpublish <package-name>@<version>
```
Comment (mikemimik) [inline docs/content/cli-commands/npm-unpublish.md]:
```suggestion
npm unpublish [<@scope>/]<pkg>@<version>
```

File: docs/content/cli-commands/npm-unpublish.md [hunk 2]
```diff
@@ -10,19 +10,21 @@ description: Remove a package from the registry
...
 ```
 
-### Warning
+#### Unpublishing an entire package
 
-**It is generally considered bad behavior to remove versions of a library
-that others are depending on!**
+```bash
+npm unpublish <package-name> -f
```
Comment (mikemimik) [inline docs/content/cli-commands/npm-unpublish.md]:
```suggestion
npm unpublish [<@scope>/]<pkg> --force
```

**Review by mikemimik — CHANGES_REQUESTED** [review]
The small inclusion of the man pages notation.

As well as the update to the command usage (`lib/unpublish.js`):
```diff
- unpublish.usage = usage('npm unpublish [<@scope>/]<pkg>[@<version>]')
+ unpublish.usage = usage(
+   'unpublish',
+   '\nnpm unpublish [<@scope>/]<pkg>[@<version>]' +
+   '\nnpm unpublish [<@scope>/]<pkg> --force'
+ )
```

**Comment by npm-deploy-user** [comment]
<table>
    <thead>
      <tr>
        <th></th>
        <th colspan="3">angular-quickstart</th>
<th colspan="3">app-large</th>
<th colspan="3">app-medium</th>
<th colspan="3">ember-quickstart</th>
<th colspan="3">react-app</th>
      </tr>
      <tr>
        <th></th>
        <th>prev</th>
<th>current</th>
<th>status</th>
<th>prev</th>
<th>current</th>
<th>status</th>
<th>prev</th>
<th>current</th>
<th>status</th>
<th>prev</th>
<th>current</th>
<th>status</th>
<th>prev</th>
<th>current</th>
<th>status</th>
      </tr>
    </thead>
    <tbody>
      <tr>
<td>initial install</td>
<td>41s</td>
<td>37.5s</td><td>✅</td>
<td>39.5s</td>
<td>36.9s</td><td>✅</td>
<td>34.5s</td>
<td>31.6s</td><td>✅</td>
<td>28.1s</td>
<td>25.6s</td><td>✅</td>
<td>33.5s</td>
<td>30.2s</td><td>✅</td>
</tr>
<tr>
<td>repeat install</td>
<td>9.6s</td>
<td>8.4s</td><td>✅</td>
<td>8.6s</td>
<td>7.6s</td><td>✅</td>
<td>8.3s</td>
<td>8.2s</td><td>✅</td>
<td>7.7s</td>
<td>7.1s</td><td>✅</td>
<td>9.1s</td>
<td>8s</td><td>✅</td>
</tr>
<tr>
<td>with warm cache</td>
<td>32.3s</td>
<td>27.5s</td><td>✅</td>
<td>33.8s</td>
<td>31.5s</td><td>✅</td>
<td>31.3s</td>
<td>27.9s</td><td>✅</td>
<td>23.4s</td>
<td>22.4s</td><td>✅</td>
<td>29.1s</td>
<td>27.7s</td><td>✅</td>
</tr>
<tr>
<td>with node_modules</td>
<td>9.2s</td>
<td>9.8s</td><td>✅🐌</td>
<td>8.6s</td>
<td>7.3s</td><td>✅</td>
<td>9s</td>
<td>7.2s</td><td>✅</td>
<td>7.8s</td>
<td>6.4s</td><td>✅</td>
<td>9.4s</td>
<td>8.7s</td><td>✅</td>
</tr>
<tr>
<td>with lockfile</td>
<td>32.6s</td>
<td>28.3s</td><td>✅</td>
<td>31s</td>
<td>29.2s</td><td>✅</td>
<td>29s</td>
<td>26.7s</td><td>✅</td>
<td>21.7s</td>
<td>20.1s</td><td>✅</td>
<td>27.4s</td>
<td>24.6s</td><td>✅</td>
</tr>
<tr>
<td>with warm cache and node_modules</td>
<td>9.3s</td>
<td>7.7s</td><td>✅</td>
<td>7.9s</td>
<td>7.3s</td><td>✅</td>
<td>8.4s</td>
<td>6.9s</td><td>✅</td>
<td>7.6s</td>
<td>6.5s</td><td>✅</td>
<td>9.2s</td>
<td>7.6s</td><td>✅</td>
</tr>
<tr>
<td>with warm cache and lockfile</td>
<td>24.7s</td>
<td>21.6s</td><td>✅</td>
<td>26.3s</td>
<td>25.1s</td><td>✅</td>
<td>24.3s</td>
<td>22.6s</td><td>✅</td>
<td>17.8s</td>
<td>14.9s</td><td>✅</td>
<td>21.3s</td>
<td>19.5s</td><td>✅</td>
</tr>
<tr>
<td>with node_modules and lockfile</td>
<td>10.1s</td>
<td>8.8s</td><td>✅</td>
<td>9.8s</td>
<td>8s</td><td>✅</td>
<td>8.6s</td>
<td>7.9s</td><td>✅</td>
<td>7.8s</td>
<td>7s</td><td>✅</td>
<td>9.7s</td>
<td>8.3s</td><td>✅</td>
</tr>
    </tbody>
  </table>

**Comment by mikemimik** [comment]
Note: will be moving this from [Sprint 2](https://github.com/npm/cli/milestone/3) -> [Sprint 3](https://github.com/npm/cli/milestone/4) and rolled into the next release.

**Comment by darcyclarke** [comment]
@nomadtechie noticed this got missed from our latest release & we figured out why; This PR has failing tests so it was skipped in the triage last week. Apologies for that. If we can fix whatever the issues are w/ linting/test coverage we'll pull this in next.

**Comment by nomadtechie** [comment]
> @nomadtechie noticed this got missed from our latest release & we figured out why; This PR has failing tests so it was skipped in the triage last week. Apologies for that. If we can fix whatever the issues are w/ linting/test coverage we'll pull this in next.

lint error is resolved now - thanks again @darcyclarke !

**Pull request closed without merging**